sunspot/lib/sunspot/schema.rb

```diff
@@ -89,7 +89,12 @@ def add_filter(filter)
     #
     def to_xml
       template = File.join(File.dirname(__FILE__), '..', '..', 'templates', 'schema.xml.erb')
-      ERB.new(File.read(template), nil, '-').result(binding)
+
+      if RUBY_VERSION >= '2.6'
+        ERB.new(File.read(template), trim_mode: '-').result(binding)
+      else
+        ERB.new(File.read(template), nil, '-').result(binding)
```

Comment:
Maybe move `.result(binding)` out of the `if` statement?

Reply:
Thanks for review. I fixed it.